$NetBSD$

Solve an infinite loop when force changed passwords is in effect
http://trac-hacks.org/ticket/3233

Index: acct_mgr/web_ui.py
===================================================================
--- acct_mgr/web_ui.py	(revision 4132)
+++ acct_mgr/web_ui.py	(working copy)
@@ -185,7 +185,7 @@
         if req.authname and req.authname != 'anonymous':
             if req.session.get('force_change_passwd', False):
                 redirect_url = req.href.prefs('account')
-                if req.path_info != redirect_url:
+                if req.href(req.path_info) != redirect_url:                
                     req.redirect(redirect_url)
         return (template, data, content_type)
 
